scripts/orig_scripts/txt2img.py

```diff
@@ -240,11 +249,17 @@ def forward(self, x, sigma, uncond, cond, cond_scale):
 
     start_code = None
     if opt.fixed_code:
-        start_code = torch.randn([opt.n_samples, opt.C, opt.H // opt.f, opt.W // opt.f], device=device)
+        shape = [opt.C, opt.H // opt.f, opt.W // opt.f]
+        if device.type == 'mps':
+            start_code = torch.randn(rand_size, device='cpu').to(device)
```

Comment:
`rand_size` is undefined here (I believe this should be updated to `shape`, and the line below)

Reply:
even after fixing that issue locally, `--fixed-code` is still broken because `opt.n_samples` is omitted from from the beginning of the shape